ern-api-impl-gen/src/generators/ApiImplGen.js

```diff
@@ -69,27 +78,39 @@ export default class ApiImplGen {
     await spin(`Downloading ${dependency.toString()}`, yarn.add(dependency.path))
   }
 
-  async getDependencies (apiDependency: Dependency) : Promise<Array<Dependency>> {
+  async getDependencies (apiDependency: Dependency): Promise<Array<Dependency>> {
     try {
       log.info(`Looking for peerDependencies`)
       const apiPackageInfo = await yarn.info(apiDependency.path, {json: true})
 
-      let dependencies = []
+      let pluginsNames = []
+
       if (apiPackageInfo.data.peerDependencies) {
-        let pluginsNames = []
-        for (let dependency in apiPackageInfo.data.peerDependencies) {
-          pluginsNames.push(`${dependency}@${apiPackageInfo.data.peerDependencies[dependency]}`)
-        }
-        dependencies = _.map(pluginsNames, Dependency.fromString)
-      } else {
-        log.warn('no peer dependencies found.')
+        this.pushDependencyNames(apiPackageInfo.data.peerDependencies, pluginsNames)
+      }
+
+      if (apiPackageInfo.data.dependencies) {
+        this.pushDependencyNames(apiPackageInfo.data.dependencies, pluginsNames)
+      }
+
+      if (pluginsNames.length === 0) {
+        log.info(`no other dependencies found for ${apiDependency.name}`)
       }
-      return dependencies
+      return _.map(pluginsNames, Dependency.fromString)
     } catch (e) {
       throw new Error(`getDependencies: ${e}`)
     }
   }
 
+  pushDependencyNames (dependencies: Object, pluginsNames: Array<string>) : Array<string> {
+    for (const dependency in dependencies) {
+      if (dependencies.hasOwnProperty(dependency)) {
```

Comment:
This is a bit hard to grasp ;) This can be simplified using `Object.keys` function instead :

> The Object.keys() method returns an array of a given object's own enumerable properties, in the same order as that provided by a for...in loop (the difference being that a for-in loop enumerates properties in the prototype chain as well).

So your `for` loop can just become 
```javascript
for (const dependency of Object.keys(dependencies))
```

And you can get rid of the conditional.